### PR TITLE
Apply session feedback screen design

### LINKED ISSUE #40: [Assigned] Session feedback screen
## Overview (Required)
- With this feature, users can send feedback for session.
- use this link https://docs.google.com/forms/d/e/1FAIpQLSdNeo0djZbTRb1toK4mIRiVcgT4Gif6YZCk5b3WYEl-c77d5w/viewform
- https://github.com/DroidKaigi/conference-app-2017/blob/master/app/src/main/java/io/github/droidkaigi/confsched2017/view/fragment/SessionFeedbackFragment.java

### PR DESCRIPTION
**Pull request by futabooo**
## Issue
- close #40 

## Overview (Required)
- Apply session feedback screen design
    - I used the layout of last year. thanks!! @konifar san

## Screenshot
 After1 | After2 | After3
:--: | :--: | :--:
 |  |

### CODE REVIEW

**Comment by takahirom** [comment]
🆒

File: app/src/main/java/io/github/droidkaigi/confsched2018/presentation/sessions/feedback/SessionsFeedbackFragment.kt [hunk 1]
```diff
@@ -28,6 +28,22 @@ class SessionsFeedbackFragment : Fragment(), Injectable {
 
     @Inject lateinit var viewModelFactory: ViewModelProvider.Factory
 
+    private val onCurrentRankingChangeListener: FeedbackRankingView.OnCurrentRankingChangeListener
```
Comment (takahirom) [inline app/src/main/java/io/github/droidkaigi/confsched2018/presentation/sessions/feedback/SessionsFeedbackFragment.kt]: 👍

**Comment by takahirom** [comment]
LGTM 💯

**Comment by futabooo** [comment]
thank you for your review 😄

**Pull request merged**